windows/hid.c

```diff
@@ -1003,6 +1030,16 @@ HID_API_EXPORT hid_device * HID_API_CALL hid_open_path(const char *path)
 	return dev;
 }
 
+static int hid_write_output_report(hid_device *dev, const unsigned char *data, size_t length)
```

Comment:
I'd prefer `hid_internal_write_output_report` - so it is easily distinguishable from public method